### DISCUSSION TITLE
New App Icon

### DISCUSSION THREAD

**Discussion by lovfi** [ente-io/ente#819] [Enhancements]
A flashy new icon for the mobile apps would be a wonderful addition and make the service seem more appealing to new users just checking things out on their native app store. 

My personal suggestion would be to implement the new ducky mascot. A plain icon of a duck giving a thumbs up would be adorable, or more practically it could be holding a photo icon. 

If the duck seems too over the top, then a nice simple photo or gallery icon is always good. I think anything more modern would be an improvement over the plain white "ente" icon that isn't very inviting.

**Comment by brog-io** [comment]
This is planned, there have been some designs for it ready not sure which one is going to be picked

**Comment by lovfi** [comment]
Ah that's good to here! Do you know where I can track the progress on this and see the current designs? I didn't see it on the roadmap so I was unaware of this all.

**Comment by brog-io** [comment]
You can see the designs on [Figma](https://www.figma.com/file/SYtMyLBs5SAOkTbfMMzhqt/ente-Visual-Design?type=design&node-id=10489-54188&mode=design), not sure why it isn't on the roadmap.

**Comment by ua741** [comment]
We now have a new icon and landing page for Ente Auth https://ente.io/auth :)
